src/legacy/core_plugins/kibana/public/management/sections/settings/components/advanced_settings_voice_announcement/advanced_settings_voice_announcement.test.tsx

```diff
@@ -22,13 +22,32 @@ import { shallow } from 'enzyme';
 
 import { AdvancedSettingsVoiceAnnouncement } from './advanced_settings_voice_announcement';
 
+const settingPartial = {
+  name: 'name',
```

Comment:
Should this be explicitly typed?

Reply:
I attempted doing this and it seemed like more work than its worth - the types are verified when the functions are called. But I might be misunderstanding something.